__integrations__/__tests__/models/lollipopkeys.test.ts

```diff
@@ -0,0 +1,166 @@
+import { createContext } from "../../utils/cosmos_utils";
```

Comment:
Add a test when a `create` is called on a document with version `1` and version `0` was deleted by cosmos.

Reply:
Done in [b869d72](https://github.com/pagopa/io-functions-lollipop/pull/19/commits/b869d72624f26db4ba5ff6ddf20e5ec2709eb6ed)